src/main/java/com/dev/cinema/validation/FieldsValueMatch.java

```diff
@@ -0,0 +1,25 @@
+package com.dev.cinema.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = FieldsValueMatchValidator.class)
+@Target({ ElementType.TYPE })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface FieldsValueMatch {
+    String message() default "Fields values don't match!";
+    String field();
+    String fieldMatch();
+    Class<?>[] groups() default {};
+    Class<? extends Payload>[] payload() default {};
+
+    @Target({ ElementType.TYPE })
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface List {
+        FieldsValueMatch[] value();
```

Comment:
You don't need this block (20-23) - you can use it when you have several pairs to check (like password - repeatpassword, email - repeatEmail, etc)